src/main/python/__init__.py

```diff
@@ -151,14 +151,15 @@ def _ppr_single_target(self, edge_type, seeds, target, relative_error=0.1, reset
 
 
     # TempestDB methods
-    def ppr(self, edge_type, seeds, num_steps=100000, reset_probability=0.3, max_results = None):
+    def ppr(self, edge_type, seeds, seed_node_type, target_node_type, num_steps=100000, reset_probability=0.3, alternating = True, max_results = None):
```

Comment:
Good idea. I'm not super-happy with the complexity--maybe the client should break this into a PPR call and a SALSA call (where SALSA does alternating walks)?